clang/test/SemaHLSL/resource_binding_attr_error_udt.hlsl

```diff
@@ -0,0 +1,131 @@
+// RUN: %clang_cc1 -triple dxil-pc-shadermodel6.3-library -x hlsl -o - -fsyntax-only %s -verify
+
+// TODO: Implement "Buffer", we use a substitute UDT
+// to test the 't' binding type for this test.
+
+template<typename T>
+struct [[hlsl::resource_class(SRV)]] MyTemplatedSRV {
+  T x;
+};

```

Comment:
Is this a case that we want to support?

For example, I think we expect typed SRV resources to be something like this:

```c++
template<typename T>
struct MyTemplatedSRV {
  __resource_handle_t [[hlsl::resource_class(srv)]] handle;
}
```

Reply:
I've removed templated SRV, because it wasn't being used by the test.

Reply:
I see that these cases got moved, I've started a new thread to discuss them.